runtime/j9vm/jvm.c

```diff
@@ -2085,7 +2092,7 @@ JNI_CreateJavaVM_impl(JavaVM **pvm, void **penv, void *vm_args, BOOLEAN isJITSer
 			}
 		}
 		if (appendToLibPath) {
-			addToLibpath("/usr/lib", FALSE);
+			addToLibpath(usrLib, FALSE);
```

Comment:
This won't compile: `usrLib` is not visible here.

Reply:
Fixed.